web/modules/custom/joinup_cck/joinup_cck.info.yml

```diff
@@ -0,0 +1,5 @@
+name: 'Joinup CCK'
```

Comment:
Having a module only to hang a script on each page looks like an arrogance to me :) What if we rename to `joinup_gdpr` so that we can drop here also other related functionality in the future (but I'm not so sure that CCK is part of GDPR, is it?).

But, as I remember, we've been discussed the alternative to provide a PR against this module https://github.com/openeuropa/oe_webtools. Wouldn't that be better?

Reply:
I renamed it to joinup_privacy as gdpr can also be quite specific. Now we can have all privacy settings in this module. 
About the second part, I do not recall something about that. Is the webtools planning to have this CCK as a default functionality? We could go with creating the PR towards them but we would need collaboration.